src/modules/authentication/authUtils.ts

```diff
@@ -20,6 +20,6 @@ export function getLoginType(): LoginType {
  * -js/blob/9274fac6d100a6300eb2faa4c94aa2431b1ca4b0/lib/msal-browser/src/utils/BrowserUtils.ts#L49
 */
 export function getCurrentUri(): string {
-  const currentUrl = window.location.href.split('?')[0].split('#')[0];
-  return currentUrl.toLowerCase();
+  const currentUrl = window.location.href.split('?')[0].split('#')[0] + 'blank.html';
```

Comment:
**question:** what is the effect of adding the blank.html to the URL?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/errors.md#monitor_window_timeout

We will no longer be getting monitor_window_timeout errors according to the msal documentation. I could use your input on this